gprofiler/profilers/java.py

```diff
@@ -846,7 +846,7 @@ def start(self) -> None:
         super().start()
         try:
             # needs to run in init net NS - see netlink_kernel_create() call on init_net in cn_init().
-            run_in_ns(["net"], lambda: proc_events.register_exit_callback(self._proc_exit_callback), 1)  # type: ignore
+            run_in_ns(["net"], lambda: cast(None, proc_events.register_exit_callback(self._proc_exit_callback)), 1)
```

Comment:
Why is it needed? o.O `run_in_ns` has type T

Reply:
Removed later

Reply:
> gprofiler/profilers/java.py:848: error: Returning Any from function declared to return "object"

I guess `T` is an `object` and it `Any` is not  
See [this issue](https://github.com/python/mypy/issues/9590)

Anyway, this code is irrelevant after merging with `master`